src/Depends.Core/DependencyAnalyzer.cs

```diff
@@ -297,6 +297,14 @@ private DependencyGraph.Builder CreateBuilder(IProjectAnalyzer  projectAnalyzer,
                     //}
                 }
 
+                // Not all dependencies are necessarily included in the list of libraries
+                // for the current target framework and runtime identifier. Add these to libraryNodes
+                // so we can still record these dependencies.
+                foreach (var dependency in libraries.SelectMany(library => library.Dependencies))
```

Comment:
Iterating over all the libraries, and it's dependencies here has some perf penalties. I mean we have just exited a `foreach` loop where we just iterated over all libraries. Can we somehow combine this with the above loop?

Reply:
Yeah, absolutely. I thought it looked cleaner as a separate step, but I can move it up into the first loop if you prefer that.